src/components/SharedComponents/SubjectPhasePicker/SubjectPhasePicker.tsx

```diff
@@ -448,9 +448,10 @@ const SubjectPhasePicker: FC<SubjectPhasePickerData> = ({
                 </Box>
                 <Box $mt={24}>
                   <OwaLink
-                    page={"oak-curriculum"}
+                    page={"curriculum-previous-downloads"}
                     $textDecoration={"underline"}
                     $font={"heading-7"}
+                    data-testid="previousPlansLink"
                   >
                     Previously released plans
                     <Icon
```

Comment:
@HelenHarlow Can you confirm what icon you would like this changed to?

Reply:
Could we use an arrow please?
<img width="96" alt="Screenshot 2024-02-20 at 16 29 10" src="https://github.com/oaknational/Oak-Web-Application/assets/144228131/f0fb0009-1772-4724-981f-c732f3796bbe">

Reply:
@lioneljingles Over to you!